dubbo-plugin/dubbo-reactive/src/test/java/org/apache/dubbo/reactive/CreateObserverAdapter.java

```diff
@@ -1,63 +0,0 @@
-/*
```

Comment:
Why remove these classes?

Reply:
When analyzing the code, it was found that `org.apache.dubbo.rpc.protocol.tri.transport.TripleHttp2FrameServerHandler` is no longer in use, so I cleaned up the related classes to facilitate code reference analysis. The test class `org.apache.dubbo.reactive.CreateObserverAdapter` referred to the unnecessary ServerCallToObserverAdapter.
When it's time for the final merge, I will split the cleanup work into a separate PR.